assets/style.css

```diff
@@ -0,0 +1,4 @@
+/* TODO // once linked from head correctly, move all css in to here */
+body {
+    border: 4px solid tomato !important; /* remove me */
```

Comment:
this is just a test style to show that's working.
Would be good to move all styles out here soon

Reply:
Deffo!